include/execution.hpp

```diff
@@ -2681,6 +2681,8 @@ namespace std::execution {
         return tag_invoke(*this, (_Scheduler&&) __sched, (_Values&&) __vals...);
       }
       template <scheduler _Scheduler, __movable_value... _Values>
+        requires (!tag_invocable<transfer_just_t, _Scheduler, _Values...> &&
+          !typed_sender<tag_invoke_result_t<transfer_just_t, _Scheduler, _Values...>>)
```

Comment:
Shouldn't this be:
```suggestion
        requires (!tag_invocable<transfer_just_t, _Scheduler, _Values...> ||
          !typed_sender<tag_invoke_result_t<transfer_just_t, _Scheduler, _Values...>>)
```

Reply:
I've missed this, thanks for noticing! Updated the commit.